app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.js

```diff
@@ -43,25 +51,51 @@ window.GOVUK.analyticsGA4.analyticsModules = window.GOVUK.analyticsGA4.analytics
         return
       }
 
-      if (this.isMailToLink(href)) {
+      var linkAttributes = element.getAttribute('data-ga4-link')
+      if (linkAttributes) {
+        linkAttributes = JSON.parse(linkAttributes)
+        clickData = Object.assign(clickData, linkAttributes)
```

Comment:
I don't believe `Object.assign` is supported by Internet Explorer. Fortunately (as long as my understanding of assign is correct), we already have a backwards-compatible script for merging two objects, see: https://github.com/alphagov/govuk_publishing_components/blob/main/spec/javascripts/govuk_publishing_components/lib/extend-object-spec.js

Reply:
Thanks @andysellick - will use that instead :+1: